internal/manager.go

```diff
@@ -148,7 +148,7 @@ func (m *Manager) LookupSdkWithInstall(name string) (*Sdk, error) {
 	}
 }
 
-func (m *Manager) LoadAllSdk() (map[string]*Sdk, error) {
+func (m *Manager) LoadAllSdk() (*AllSdk, error) {
```

Comment:
I took a closer look, we just change the return value from `map[string]*Sdk` to `[]*Sdk`.  

Current change is a bit complicated.

Reply:
Okay, I will revise it later.